h2o-r/h2o-DESCRIPTION.template

```diff
@@ -103,3 +108,4 @@ Collate:
          'zzz.R'
```

Comment:
I'm wondering if the `zzz.R` shouldn't be the last file in the list.

Reply:
Yeah, let's move it to the end (it used to be there).

Reply:
Yes, let's move it there, thanks!